tensorflow_recommenders_addons/dynamic_embedding/core/utils/utils.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TFRA_UTILS_H_
+#define TFRA_UTILS_H_
+
+namespace tensorflow {
+namespace recommenders_addons {
+
+#define CONCAT(X, Y, Z) (#X #Y #Z)
```

Comment:
undef `CONCAT` to avoid macro conflict, since it only used to get `DECORATE_OP_NAME`